### PR TITLE
Avoid lodash where possible

### PR DESCRIPTION
**Pull request by TrySound**
In this diff I replaced some lodash methods with native ones without
complicating code.

(Related PR: #393)

### CODE REVIEW

**Comment by TrySound** [comment]
@th0r Is this better?

**Comment by valscion** [comment]
I would like to :+1: this idea of reducing the surface area of `lodash` in favor of native ones. I have had trouble deciphering what the lodash methods do as I'm myself more accustomed to the native methods.

File: src/analyzer.js [hunk 1]
```diff
@@ -169,7 +168,7 @@ function readStatsFromFile(filename) {
 }
 
 function getChildAssetBundles(bundleStats, assetName) {
-  return _.find(bundleStats.children, (c) =>
+  return bundleStats.children.find((c) =>
```
Comment (th0r) [inline src/analyzer.js:171]: Are you sure `bundleStats.children` is always defined?

File: src/parseUtils.js [hunk 2]
```diff
@@ -342,9 +342,10 @@ function getModulesLocations(node) {
       node.arguments[0].elements :
       node.elements;
 
-    return _.transform(modulesNodes, (result, moduleNode, i) => {
-      if (!moduleNode) return;
+    return modulesNodes.reduce((result, moduleNode, i) => {
+      if (!moduleNode) return result;
```
Comment (th0r) [inline src/parseUtils.js:346]: Let's better inverse condition and return only once in the end of the function please.

File: src/utils.js [hunk 3]
```diff
@@ -15,11 +15,11 @@ function createAssetsFilter(excludePatterns) {
         pattern = new RegExp(pattern, 'u');
       }
 
-      if (_.isRegExp(pattern)) {
+      if (types.isRegExp(pattern)) {
```
Comment (th0r) [inline src/utils.js:18]: What is `types`?

**Review by th0r — CHANGES_REQUESTED** [review]

File: src/utils.js (hunk 3 above)
Comment (TrySound) [reply src/utils.js:18]: New node.js utilities
https://nodejs.org/api/util.html#util_util_types

File: src/analyzer.js (hunk 1 above)
Comment (TrySound) [reply src/analyzer.js:171]: Nope, relied on tests. Will coalesce with []

File: src/utils.js (hunk 3 above)
Comment (th0r) [reply src/utils.js:18]: Nice, didn't know about it.

**Comment by TrySound** [comment]
Hi @th0r is this good now?

**Comment by th0r** [comment]
Yep, thanks!

**Pull request merged**